build.sbt

```diff
@@ -35,14 +35,6 @@ val commonSettings = Seq(
   Test / javaOptions += "-Dconfig.resource=TEST.public.conf"
 ) ++ buildInfoSettings
 
-lazy val dynamoDBLocalSettings = Seq(
```

Comment:
Another benefit is that we no longer need to run the local DynamoDB tests which were quite flakey